dfastmi/io/map_file.py

```diff
@@ -36,6 +37,11 @@
 FACE_LOCATION = "face"
 
 
+class FileType(Enum):
```

Comment:
why an enum and not 2 types implementing specific code for the 2 based on a common base type?